USER_MANUAL.md

```diff
@@ -628,7 +673,526 @@ The measured total site level emission rate on January 30th is assumed to have l
+
+**Description:** The number of days required to have passed since the first site added to the site watch list before a site can be flagged. The company will hold all measurements in a site watch list. The emissions rate used to triage flagging based on follow-up threshold and proportion are specified with [redundancy filter](#redundancy_filter).
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** N/A
+
+#### &lt;instant_threshold&gt;
+
+**Data type:** Numeric (Float)
+
+**Default input:** 0.0
+
+**Description:** The follow-up instant threshold in grams per second. Measured site-level emissions must be above the follow-up threshold before a candidate site becomes immediately available for flagging. If _follow_up.instant_threshold_type_ is "absolute", the numeric value indicates the follow-up threshold in grams per second. If "relative", the numeric value is passed to a function that calculates the emission rate that corresponds to a desired proportion of total emissions for a given leak size distribution. The function estimates the MDL needed to find the top X percent of sources for a given leak size distribution. For example, given a proportion of 0.01 and a leak-size distribution, this function will return an estimate of the follow-up threshold that will ensure that all leaks in the top 1% of leak sizes are found.
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** Follow-up thresholds are explored in detail in Fox et al. 2021\. Choosing follow-up rules is complex and work practices should be developed following extensive analysis of different scenarios. It is important to understand how follow-up thresholds and follow-up ratios interact, especially if both are to be used in the same program. Note that follow-up thresholds are similar to minimum detection limits but that the former is checked against the measured emission rate (which is a function of quantification error) while the latter is checked against the true emission rate.
+
+#### &lt;interaction_priority&gt;
+
+**Data type:** String
+
+**Default input:** "threshold"
+
+**Description:** Specifies which algorithm to run first on candidate sites when determining which to flag. If the value is _threshold_ the proportion of sites to follow up with will be taken from all sites over the threshold. If the value is _proportion_ the proportion of sites will be taken from the candidate sites, then from those sites follow-up will occur at sites above the threshold.
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** N/A
+
+#### &lt;proportion&gt;
+
+**Data type:** Numeric
+
+**Default input:** 1.0
+
+**Description:** A single value that defines the proportion of candidate flags to receive follow-up. For example, if the follow-up ratio is 0.5, the top 50% of candidate flags (ranked by measured emission rate) will receive follow-up. Candidate flags have already been checked against the minimum detection limit.
```

Comment:
Can we revisit this description to see if we can update to be easier to understand?